demo/README.md

```diff
@@ -17,9 +17,9 @@ in the corresponding folder.
 About Tycho
 ===========
 
-  * [Project Homepage](http://www.eclipse.org/tycho/)
-  * [Documentation](http://eclipse.org/tycho/documentation.php)
-  * [Bug Tracker](https://bugs.eclipse.org/bugs/buglist.cgi?product=Tycho)
-  * [How to Contribute](http://wiki.eclipse.org/Tycho/Contributor_Guide)
+  * [Project Homepage](https://github.com/eclipse/tycho)
+  * [Documentation](https://www.eclipse.org/tycho/sitedocs/)
+  * [Bug Tracker](https://github.com/eclipse/tycho/issues)
+  * [How to Contribute](https://github.com/eclipse/tycho/blob/master/CONTRIBUTING.md)
```

Comment:
You could probably replace the full URL here (and in other cases) with a [relative link](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files). That would then also work properly for other branches, linking to the respective file of that branch, instead of `master`.
And despite not explicitly being mentioned in the documentation, it looks like starting the link with `/` allows linking relative to the repository root; this would avoid having to use `..` to navigate the file structure.